src/main/java/com/swervedrivespecialties/exampleswerve/constants/ChassisConstants.java

```diff
@@ -0,0 +1,35 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package com.swervedrivespecialties.exampleswerve.constants;
+
+import org.frcteam2910.common.control.PidConstants;
+
+/**
+ * Add your docs here.
+ */
+public class ChassisConstants {
+
+    public static final double TRACKWIDTH = 19.5;
+    public static final double WHEELBASE = 23.5;
+
+    public static final double FRONT_LEFT_ANGLE_OFFSET = -Math.toRadians(0.0);
+    public static final double FRONT_RIGHT_ANGLE_OFFSET = -Math.toRadians(0.0);
+    public static final double BACK_LEFT_ANGLE_OFFSET = -Math.toRadians(0.0);
+    public static final double BACK_RIGHT_ANGLE_OFFSET = -Math.toRadians(0.0);
+
+    //These will probably need to be changed for the actual robot
+    public static final PidConstants ANGLE_CONSTANTS = new PidConstants(0.5, 0.0, 0.0001); //Angle constants, angle reduction, drive reduction and wheel diameter are defaults
+    public static final double ANGLE_REDUCTION = 18.0/1.0;
```

Comment:
add whitespace around your binary operators on each side, or don't. (I mean, you should, but a few morons argue you shouldn't). Whatever you do, be consistent. make it 18.0 / 1.0 and (1 - INITIAL_SPEED_SCALE) / (NUM_SPEEDS - 1) or make it 18.0/1.0 and (1-INITIAL_SPEED_SCALE)/(NUM_SPEEDS-1)